api/.nycrc.json

```
@@ -0,0 +1,10 @@
+{
+  "all": true,
+  "reporter": "html",
+  "include": [
+    "lib/**/*.js"
+  ],
+  "exclude": [
+    "eslintrc.js"
+  ]
```

Une syntaxe qui fonctionne pour exclure tous les fichiers `.eslintrc.js` est la suivante et cela permet de ne pas avoir à ajouter ces instructions `/* istanbul ignore file */` qu'on trouve dans la PR et qu'on peut oublier.

```suggestion
  "exclude": [
    "**/.eslintrc.js"
  ]
```

---

C'est plus élégant. Corrigé